ee/debug/checkups/runtime.go

```diff
@@ -118,6 +118,12 @@ func gatherPprofCpu(z *zip.Writer) error {
 		return fmt.Errorf("creating cpuprofile: %w", err)
 	}
 
+	// Increase the CPU profiling rate, in the hopes we can capture more / better detail. This
+	// is quite unclear, there's a long discussion at https://github.com/golang/go/issues/40094 and
+	// linked issues. But my takeaway is that in _some_ cases, there's value in raising this. But it
+	// does produce a spurious error -- "runtime: cannot set cpu profile rate until previous profile has finished"
+	runtime.SetCPUProfileRate(200)
```

Comment:
I think this might actually have a detrimental effect on Windows -- https://github.com/golang/go/issues/61665 (especially https://github.com/golang/go/issues/61665#issuecomment-1659714314)

Reply:
(Thank you)